### PR TITLE
OCPBUGS-20037: Fix journal entries to belong to greenboot unit

### PR DESCRIPTION
**Pull request by pacevedom**
**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

File: packaging/greenboot/microshift-running-check.sh [hunk 1]
```diff
@@ -19,6 +19,7 @@ trap 'script_exit' EXIT
 # The term handler to override the default behavior and have a uniform and
 # homogeneous exit code in all controlled situations.
 function return_failure() {
+    echo "Signal received, terminating."
```
Comment (ggiguash) [inline packaging/greenboot/microshift-running-check.sh:22]: From the handler name, it looks like it's also invoked on "return 1" from functions. 
I'm not sure it's an appropriate message in all cases.

**Review by dhellmann — COMMENTED** [review]
It would be good to have a test case for this change.

**Comment by pacevedom** [comment]
/hold
Repurposing the bug because more failures were found

File: packaging/greenboot/functions.sh [hunk 2]
```diff
@@ -145,7 +156,7 @@ function namespace_pods_ready() {
 
     # Terminate the script in case more pods are ready than expected - nothing to wait for
     if [ "${tcount}" -gt "${ct}" ] ; then
-        echo "The number of ready pods in the '${ns}' namespace is greater than the expected '${ct}' count. Terminating..."
+        echo "The number of ready pods in the '${ns}' namespace is greater than the expected '${ct}' count. Terminating..." | tee >(systemd-cat -t "${SCRIPT_NAME}")
```
Comment (dhellmann) [inline packaging/greenboot/functions.sh:159]: Can you add a comment explaining why we need this here for this echo, but not for any of the other echo calls?

Comment (pacevedom) [reply packaging/greenboot/functions.sh:159]: I included a comment in the same file, above, as its something general: https://github.com/openshift/microshift/pull/2430/files#diff-63bfc037903c0ae357d2459b936ee63776f356d70dd3c4921e2d8e0608009770R13-R21

Comment (dhellmann) [reply packaging/greenboot/functions.sh:159]: Is it possible to do this at the point where this function is called, instead of inside the function?

**Pull request merged**